packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -27,7 +27,7 @@
     "ajv": "^6.1.1",
     "eslint": "4.16.0",
     "eslint-plugin-lwc": "0.3.2",
-    "lwc-language-server": "0.16.4",
+    "lwc-language-server": "1.4.0",
```

Comment:
@rsalvador 

Will you be publishing a new version for 214/patch? Here are the versions I see today:

```
nchen@nchen-wsm1:~/Development/Node/salesforcedx-vscode|⇒  npm view lwc-language-server versions                         ± nick/swap-lwc-next-to-lwc-current ✔ 9:42:38

[ '0.0.1-java',
  '0.0.2-java',
  '0.0.3-java',
  '0.10.0',
  '0.11.0',
  '0.12.0',
  '0.13.0',
  '0.14.0',
  '0.14.1',
  '0.14.2',
  '0.15.1',
  '0.16.0',
  '0.16.1',
  '0.16.2',
  '0.16.3',
  '0.16.4',
  '1.0.0',
  '1.0.1',
  '1.0.2',
  '1.1.0',
  '1.1.1',
  '1.2.0',
  '1.2.1',
  '1.3.0',
  '1.3.1',
  '1.4.0' ]
```

Reply:
I'll publish a new version today

Reply:
@vazexqi just published lwc-language-server:1.5.0, please use this one